sklearn/tree/tree.py

```diff
@@ -1114,12 +1124,18 @@ class DecisionTreeRegressor(RegressorMixin, BaseDecisionTree):
     reduce memory consumption, the complexity and size of the trees should be
     controlled by setting those parameter values.
 
-    The features are always randomly permuted at each split. Therefore,
-    the best found split may vary, even with the same training data and
-    ``max_features=n_features``, if the improvement of the criterion is
-    identical for several splits enumerated during the search of the best
-    split. To obtain a deterministic behaviour during fitting,
-    ``random_state`` has to be fixed.
+    The features are always randomly permuted at each split, even if 
+    ``splitter`` is set to ``"best"``. Especially if 
+    ``max_features < n_features``, the algorithm will select ``max_features`` 
+    at random at each split. But the best found split may vary accross different 
+    runs, even if ``max_features=n_features``. That is the case, if the 
+    improvement of the criterion is identical for several splits and one split
+    has to be selected at random. 
+    To obtain a deterministic behaviour during fitting, ``random_state`` has to 
+    be fixed. To fix ``random_state`` it has to be set to an arbitrary integer.  
+    The specific value does not matter here. The integer is just the seed for the 
+    random number generator. It ensures that the results will be reproducible 
+    (deterministic).
```

Comment:
same here

Reply:
Thanks for your review adrinjalali. Sorry for the spelling mistake and sorry if this is a dumb question: do I have to commit your changes to my fork or is GitHub including these changes automatically into the PR once I resolve the conversation?

Reply:
Don't worry, these things can be confusing, there are no dumb questions :)

Resolving the conversation doesn't actually change anything in the code. You need to apply the changes in your local branch, commit, and push to your fork, and then they're automatically added to this PR.